### PR TITLE
Mention that top-level and class vars are lazily inited

### LINKED ISSUE #2493: Add information about lazy initialization of global and static variables
It would be nice to add some words to the language tour about the fact that global and static variables are initialized lazy,

### PR DESCRIPTION
**Pull request by kwalrath**
Fixes #2493

### CODE REVIEW

File: src/_guides/language/language-tour.md [hunk 1]
```diff
@@ -377,7 +377,10 @@ if (weLikeToCount) {
 print(lineCount);
 ```
 
-<!-- TODO: Point to coverage of ! and late -->
+Top-level and class variables are lazily initialized;
+the initialization code runs
+the first time the variable is used.
```
Comment (parlough) [inline src/_guides/language/language-tour.md:382]: I'm not 100% sure or I may be misunderstanding, but I believe only static class variables are lazily initialized. Unless they are marked `late`. 

For example(https://dartpad.dev/?null_safety=true&id=88b7f2681551b90bfedfd15bcfc403c9):

```dart
class Test {
  final test = print(3);
}

void main() {
  Test();
}
```

prints 3.

Comment (kwalrath) [reply src/_guides/language/language-tour.md:382]: I've been using "class variables" to mean "static variables" (as opposed to instance variables), but maybe that language is olde fashioned.

Comment (parlough) [reply src/_guides/language/language-tour.md:382]: Ah yeah, that makes sense, I was mixing up my terminology. I think this should be fine then as it's actually the term used in the spec.

**Review by parlough — APPROVED** [review]
Looks good to me, thanks :D

**Pull request merged**